components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,276 +15,334 @@ gh_repo = "automate"
+    ```
 
-9. Deploy using
+8. Edit `config.toml` and add the following:
+
+    - Update the `instance_count`
+    - fqdn : load balance URL, which points to the frontend node.
+    - keys : ssh username and private keys
+    - Ensure to provide Chef backend's frontend server IPs for Automate HA Chef Automate and Chef Server.
+    - Ensure to provide Chef backend's backend server IPs for Automate HA Postgres and OpenSearch machines.
+    - Sample configuration; please modify according to your needs.
+
+    ```cmd
+    [architecture.existing_infra]
+    secrets_key_file = "/hab/a2_deploy_workspace/secrets.key"
+    secrets_store_file = "/hab/a2_deploy_workspace/secrets.json"
+    architecture = "existing_nodes"
+    workspace_path = "/hab/a2_deploy_workspace"
+    ssh_user = "myusername"
+    ssh_port = "22"
+    ssh_key_file = "~/.ssh/mykey.pem"
+    sudo_password = ""
+    # DON'T MODIFY THE BELOW LINE (backup_mount)
+    backup_mount = "/mnt/automate_backups"
+    # Eg.: backup_config = "object_storage" or "file_system"
+    backup_config = "file_system"
+    # If backup_config = "object_storage" fill out [object_storage.config] as well
+    ## Object storage similar to AWS S3 Bucket
+    [object_storage.config]
+    bucket_name = ""    
+    access_key = ""
+    secret_key = ""
+    # For S3 bucket, default endpoint value is "https://s3.amazonaws.com"
+    # Include protocol to the enpoint value. Eg: https://customdns1.com or http://customdns2.com
+    endpoint = ""
+    # [Optional] Mention object_storage region if applicable
+    # Eg: region = "us-west-1"
+    region = ""
+    ## === ===
+    [automate.config]
+    # admin_password = ""
+    # automate load balancer fqdn IP or path
+    fqdn = "chef.example.com"
+    instance_count = "2"
+    # teams_port = ""
+    config_file = "configs/automate.toml"
+    [chef_server.config]
+    instance_count = "2"
+    [opensearch.config]
+    instance_count = "3"
+    [postgresql.config]
+    instance_count = "3"
+    [existing_infra.config]
+    automate_private_ips = ["10.0.1.0","10.0.2.0"]
+    chef_server_private_ips = ["10.0.1.0","10.0.2.0"]
+    opensearch_private_ips = ["10.0.3.0","10.0.4.0","10.0.5.0"]
+    postgresql_private_ips = ["10.0.3.0","10.0.4.0","10.0.5.0"]
+    ```
+
+9. Deploy using the following command:
 
 ```cmd
-./chef-automate deploy config.toml <airgapped bundle name>
+./chef-automate deploy config.toml --airgap-bundle <airgapped bundle name>
 ```
 
-10. Clean up the old packages from chef-backend (like Elasticsearch and postgres)
-
-11. [Doc: Restore Backed Up Data to Chef Automate HA](/automate/ha_chef_backend_to_automate_ha/#restore-backed-up-data-to-chef-automate-ha)
+10. Clean up the old packages from the chef-backend (like Elasticsearch and Postgres)
 
+11. [Restore Backed Up Data to Chef Automate HA](/automate/ha_chef_backend_to_automate_ha/#restore-backed-up-data-to-chef-automate-ha)
 
 ## Using Automate HA for Chef-Backend user
```

Comment:
This title reads confusing for me. @dishanktiwari2501

Reply:
We need to add a section to modify existing nodes to use the newly deployed and migrated Chef Automate HA instance. 
@YashviJain01  @shanmugapriya-tr

Reply:
The steps are mentioned below. DO we need any change in that @kalroy